test/recipes/95-test_external_gost_engine_data/gost_engine.sh

```diff
@@ -12,14 +12,21 @@
 #
 set -e
 
-O_EXE=`pwd`/$BLDTOP/apps
-O_BINC=`pwd`/$BLDTOP/include
-O_SINC=`pwd`/$SRCTOP/include
-O_LIB=`pwd`/$BLDTOP
+PWD="$(pwd)"
 
-export PATH=$O_EXE:$PATH
-export LD_LIBRARY_PATH=$O_LIB:$LD_LIBRARY_PATH
-export OPENSSL_ROOT_DIR=$O_LIB
+O_EXE="$PWD/$BLDTOP/apps"
+O_BINC="$PWD/$BLDTOP/include"
+O_SINC="$PWD/$SRCTOP/include"
+O_LIB="$PWD/$BLDTOP"
```

Comment:
Ouch.  If any of those are absolute paths, the failure will be glorious.  I'd suggest this instead:

``` bash
SRCTOP=$(cd $SRCTOP; pwd)
BLDTOP=$(cd $BLDTOP; pwd)
O_EXE=$BLDTOP/apps
O_BINC=$BLDTOP/include
O_SINC=$SRCTOP/include
O_LIB=$BLDTOP
```